django_freeradius/base/models.py

```diff
@@ -796,6 +777,29 @@ def prefix_add(self, prefix, n, password_length=BATCH_DEFAULT_PASSWORD_LENGTH):
         self.full_clean()
         self.save()
 
+    def get_or_create_user(self, row, users_list, generated_passwords, password_length):
+        User = get_user_model()
```

Comment:
can we move this at the top of the file to avoid it redefine it each time? The `User` model is always only 1.